### ISSUE TITLE
[EN][Love is in the Air][NPC][Minor]Kwee Q. Peddlefeet <Crown Chemical Co.>(ID 38040)

### ISSUE THREAD

**Issue by Aival94** [Atlantiss/BugTracker#6535]
There are 2 npcs spawned in exodar.

Wowhead Link: [Kwee Q. Peddlefeet <Crown Chemical Co.>](http://www.wowhead.com/npc=38040/kwee-q-peddlefeet)

**Comment by tomatosz** [comment]
Posted a fix on the forums:

https://community.atlantiss.eu/index.php?/topic/18164-npc-kwee-q-peddlefeet/

**Comment by TranceAccount** [comment]
Phasing issue? 📦

**Comment by tomatosz** [comment]
nope, same phase, just double spawn for whatever reason. (and with some other slightly different values, interestingly enough).
